src/coreclr/inc/utilcode.h

```diff
@@ -3402,6 +3402,14 @@ inline bool FitsInRel28(INT64 val64)
     return (val64 >= -0x08000000LL) && (val64 < 0x08000000LL);
 }
 
+//*****************************************************************************
+// Returns whether the offset fits into a RISC-V auipc + I-type or S-type instruction combo
+//*****************************************************************************
+inline bool FitsInAuipcCombo(INT64 val64)
```

Comment:
Nit: The Get and Put are 'GetRiscV64AuipcCombo' but this is just 'AuipcCombo', maybe make it consistent